packages/core/src/components/fields/SchemaField.js

```diff
@@ -49,6 +48,9 @@ function getFieldComponent(schema, uiSchema, idSchema, fields) {
   return componentName in fields
     ? fields[componentName]
     : () => {
+        const { fields } = registry;
```

Comment:
Why use `registry.fields.UnsupportedField` instead of using `fields.UnsupportedField`? I'm trying to see why UnsupportedField requires pulling from the registry while the other fields don't.

Reply:
Ha, well this is embarassing.